packages/shylu/shylu_dd/frosch/test/Overlap/ParameterLists/ParameterList_OneLevelPreconditioner_Graph.xml

```diff
@@ -24,6 +24,8 @@
             
             <ParameterList name="AlgebraicOverlappingOperator">
                 <Parameter name="Adding Layers Strategy"                        type="string"   value="CrsGraph"/>
+                <Parameter name="Combine Values in Overlap"                     type="string" value="Restricted"/><!--Restricted,Averaging,Full-->
+                <Parameter name="HarmonicOnOverlap"                             type="bool"   value="0"/>
```

Comment:
Could we instead change "AlgebraicOverlappingOperator" to "HarmonicOverlappingOperator"?